core/controllers/reader.py

```diff
@@ -55,6 +55,21 @@
 def require_playable(handler):
     """Decorator that checks if the user can play the given exploration."""
     def test_can_play(self, exploration_id, **kwargs):
+        """Tests if the user can play the exploration.
```

Comment:
Actually, on second thoughts, I think it'll be good (aka non-optional) to consistently leave space between paragraphs.